src/serlo-editor-repo/plugin-text/components/math-element.tsx

```diff
@@ -37,6 +37,20 @@ export function MathElement({
   const selected = useSelected()
   const preferences = useContext(PreferenceContext)
 
+  const isInsideListElement = useMemo(() => {
```

Comment:
Maybe you can just use `isOrderedListActive(editor) || isUnorderedListActive(editor)` here instead? You can get those helpers from `'../utils/list'`.